src/controllers/EventController.ts

```diff
@@ -49,6 +59,7 @@ export class EventController {
   }
 
   @Get('/')
+  @UseBefore(AuthenticationMiddleware, OfficerAuthorizationFactory)
```

Comment:
2 things:

1. Don't think we need AuthenticationMiddleware here because it has authorizationalready
2. The factory is a function that returns a middleware, so I think you just need to call it

great work!

Reply:
Should we even have authentication middleware anymore? I mean it's basically already in authorization middleware...

Reply:
guess not :p

Reply:
Maybe I'll just leave it there, whatever use case needs authentication but not authorization can use it (though I know of no such use case rn).

Reply:
Actually I just removed the authentication folder lol